conf/application.conf

```diff
@@ -172,13 +172,6 @@ contact-frontend {
   host = "http://localhost:9250"
 }
 
-delegated-service {
-  ated = {
-    sign-out-url = "http://localhost:9916/ated/logout"
-    beta-feedback-url = "http://localhost:9916/ated/feedback"
```

Comment:
I believe that the only entry that should have been removed is the beta-feedback-url ONLY
sign-out-url is called in ApplicationConfig where servicesSignOutUrl references delegate-services

Reply:
added sign-out-url back in under delegated-service